blockchain/README.md

```diff
@@ -28,17 +28,28 @@
 
 ## Files 
 
-* truffle-config.js: Truffle configuration file for setting network information and other project-related settings.
+* truffle-config.js: Truffle configuration file for setting network information and other project-related settings like compiler type (we are using version 0.8.10 atm).
 
 * contracts/Migrations.sol: Manages and updates the status of the deployed smart contract. This file comes with every Truffle project, and is **usually not edited**.
 
-* contracts/TokenFactory.sol: Needs to be deployed first, and address of smart contract copied
+* contracts/TokenFactory.sol: Needs to be deployed first. Handles token minting and is called from MainContract.
 
-* contracts/MainContract.sol: Holds the main functionality, needs to be deployed 2nd, than function setFactoryInterface() with just-copied address as parameter
+* contracts/MainContract.sol: Needs to be deployed second. Holds all main functionalities, like drop creation, joining a drop, getting drop data, etc.
 
 * migrations/1_initial_migration.js: Migration (deployment) script for the Migrations contract found in the Migrations.sol file.
 
-* migrations/2_hello_world.js: Migration script for the HelloWorld contract. (Migration scripts are run in order, so the file beginning with 2 will be run after the file beginning with 1.)
+* migrations/2_deploy_token_factory.js: Migration script for the TokenFactory contract. (Migration scripts are run in order, so the file beginning with 2 will be run after the file beginning with 1.)
+
+* migrations/3_deploy_main.js: Migration script for the main contract.
+
+
+## Technical documentation
+
+Both contracts are currently deployed on the kovan ethereum testnet. We decided to use kovan since it is not as vulnerable to spam attacks as other alternatives and has been online long enough for having a rather large community and therefore a lot of support. 
```

Comment:
Maybe also add something like "can easily be deployed to ethereum mainnet or other testnets."

Reply:
done :)